library.properties

```diff
@@ -1,10 +1,10 @@
 name=Ubidots
-version=2.1.14
+version=3.0
 license=MIT
 author=Ubidots <devel@ubidots.com>, Jose Garcia <jose.reyes@ubidots.com>
 maintainer=Jose García <jose.reyes@ubidots.com>
 url=https://github.com/ubidots/ubidots-particle
-sentence=Ubidots library for Particle
-paragraph=Intended to send or retrieve data from Ubidots through TCP or UDP using variable labels or variables ID.
+sentence=Ubidots library for Particle devices
+paragraph=Intended to send or retrieve data from Ubidots through TCP/UDP/HTTP/Webhooks.
```

Comment:
Change to TCP/UDP/HTTP/ParticleWebhooks.

Reply:
solved